src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -240,68 +241,72 @@ const ContentListTable = () => {
   } = data;
 
   const rowActions = useCallback(
-    (rowData: ContentItem): IAction[] =>
-      isRedHatRepository
-        ? features?.snapshots?.accessible
-          ? [
+    (rowData: ContentItem, repositoryUUID: string): IAction[] => {
```

Comment:
This method shouldn't be taking in repositoryUUID, instead it should use `rowData?.uuid` exactly like the deleteAction is doing a few lines below.  That was the error you were getting earlier complaining about the number of arguments